CONTRIBUTING.md

```diff
@@ -1,4 +1,44 @@
-# CONTRIBUTING
+## Guidelines
+
+Thank you for taking an interest in contributing to Qwik! 
+We appreciate you 🫶🏽  
+Below are the guidelines how to help in the best possible way.
+
+### Submitting an Issue
+
+Before creating a new issue, please try searching through open issues using the [GitHub issue search bar](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests). You might find the solution to your problem or you could verify that it is an already known problem.
+
+We want a bug-free and best-performing project, that's why we take all reported issues to heart, but if we can't reproduce the issue, we won't have a way of locating and properly fixing it.
+
+Therefore, to solve the problem in the best possible way, please create a minimum repository that reproduces the problem itself, without any extra cruft, and with the least possible code explaining the error.
+
+Without enough information to reproduce the issue, we will close it because we can't recreate and solve it.
+
+### Submitting a Pull Request (PR)
+
+#### Branch Organization
+
+We adopt [trunk-based development](https://trunkbaseddevelopment.com/) therefore all Pull Requests are made to the main branch because we do not use separate branches for development or for the versions we release.
+
+#### Good first issue
+
+The issues marked with [_Good first issue_](https://github.com/BuilderIO/qwik/issues?q=is:open+is:issue+label:%22good+first+issue%22) are a good starting point to familiarize yourself with the project.
+
+Before solving the problem, please check with the maintainers that the issue is still relevant. Please feel free to leave a comment on the issue to show your intention to work on it and prevent other people from accidentally duplicating your effort.
+
+#### Sending a Pull Request
+
+Before submitting a pull request, please consider the following guidelines:
+
+- fork the repository into your own account
```

Comment:
Because each item in this list is a complete sentence, you could start each with a capital letter and end each with a period. Totally up to you, but if you decide to edit them, they'd be like this:
```suggestion
- Fork the repository into your own account.
```

Reply:
Thanks @kapunahelewong for reviewing this PR 👍

Reply:
I added capital letters and added a period in the items without an ending command